MetaMask/metamask-extension
# Adding recovery phrase video to onboarding process

Related: https://github.com/MetaMask/metamask-extension/issues/10244

<img width="770" alt="Screen Shot 2021-03-25 at 2 47 23 AM" src="https://user-images.githubusercontent.com/8732757/112453020-7b27cb80-8d14-11eb-80f1-bbe48c585a3a.png">

Manual testing steps:  
  - Clean install MetaMask
  - Select the "Create Account Route"
  - Confirm after creating a password you are directed to the seed phrase video page
  - Confirm that clicking next further continues on to the reveal seed phrase page
  - Confirm that this component is not shown in the "Import Account flow" on fresh install"
